inc/colour-branding-import.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+function hale_get_colours_from_jason() {
+	$jason_file = get_theme_mod("customizer_setting_json");
+	if ($jason_file) {
+		$jason_string = file_get_contents($jason_file);
+		$jason = json_decode($jason_string, true);
+		$colour_array = hale_get_colours() or die("no colour array");
+
+		for($i=0;$i<count($colour_array);$i++) {
+			$colour_id = hale_get_colour_id($colour_array[$i]);
+			$jason_colour = $jason[$colour_id];
+			if (get_theme_mod($colour_id) != $jason_colour) //if JSON colour is different from current colour
```

Comment:
It it worth doing a validation here to make sure its a valid hex colour?

Reply:
No.  
- Wordpress does this check for us.  
- One of the values is not a hex value, but can be raw CSS.  We should remove this eventually but for now it has to be there.